test/ResourcesClassLibraryNoAttribute/ResourcesClassLibraryNoAttribute.csproj

```diff
@@ -3,8 +3,7 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net452;netstandard1.6</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.6</TargetFrameworks>
+    <TargetFramework>netstandard1.0</TargetFramework>
```

Comment:
I assume that in these two cases where we went down some versions that the standard was "lowest one that works"?